File: Capítulo_4/Pilhas/Criando uma classe JavaScript Stack baseada Obj/Push de elementos na pilha/Código em Python/push-pilha.py
class Stack:
    def __init__(self):
        self.count = 0
        self.items = {}
    
    def push(self, element):
        self.items[self.count] = element
        self.count += 1

stack = Stack()
stack.push(5)
stack.push(8)

print(stack.items)
print(stack.count)

File: Capítulo_4/Pilhas/Criando uma classe JavaScript Stack baseada Obj/Push de elementos na pilha/push-pilha.js
// Como estamos trabalhando com um objeto, essa versão do método push nos permite fazer push somente de um único elemento de cada vez. Podemos ver o código do método push a seguir:
class Stack {
    constructor() {
        this.count = 0;
        this.items = {};
    }
    push(element) {
        this.items[this.count] = element;
        this.count++;
    }
}

const stack = new Stack();
stack.push(5);
stack.push(8);

console.log(stack.items);
console.log(stack.count);